Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -189,15 +217,24 @@ namespace Multiplayer
 
                 if (lostInput)
                 {
+                    //TODO: Maybe add to Audit Trail here (server)
```

Comment:
Consider migrating TODOs to public GitHub issues. TODOs are hard for other developers to take action on as its unclear who is doing the work or why